src/main/java/org/opentripplanner/middleware/models/ItineraryExistence.java

```diff
@@ -95,6 +97,41 @@ public ItineraryExistenceResult getResultForDayOfWeek(DayOfWeek dayOfWeek) {
         throw new IllegalArgumentException("Invalid day of week provided!");
     }
 
+    /**
+     * Helper function to adapt OTP request parameters to GraphQL's JSON {@code variables} object, limited to
+     * variables specified in the GraphQL plan.  Parameter values are all simply Strings, so we can't use
+     * {@code ObjectMapper} to get the types right, must hardcode them specifically for ItineraryExistence so
+     * that the values that are actually strings have {@code \"} around them.
+     */
+    private static String paramsToVariables(Map<String, String> params) {
+        StringBuilder builder = new StringBuilder("{");
+        params.forEach((k, v) -> {
+            switch (k) {
+                case "arriveBy":
+                case "numItineraries":
+                    builder.append("\"" + k + "\":" + v + ",");
```

Comment:
Trailing commas will result in request failure.

Reply:
Fixed in last commit.